### PR TITLE
fix login issues

### PR DESCRIPTION
**Pull request by Can-Tina**

### CODE REVIEW

File: src/client/pages/signIn/SignIn.jsx [hunk 1]
```diff
@@ -22,10 +25,12 @@ const SignIn = () => {
...
         })
         const data = await res.json();
-        //check data for error
-        //if error, return false
-        //else localStorage.setItem('auth', data.data) and return true
-        return data;
+        if (data.error) {
+            return false;
+        }
+        console.log(data)
```
Comment (alexamaingard) [inline src/client/pages/signIn/SignIn.jsx:31]: Remove printing statement please :)

**Review by alexamaingard — APPROVED** [review]

File: src/client/pages/signIn/SignIn.jsx (hunk 1 above)
Comment (glowkeeper) [reply src/client/pages/signIn/SignIn.jsx:31]: +1 (although I'm not going to let that stop me approving) - but _don't_ spam the console!

File: src/server/controllers/user.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ const createUser = async (req, res) => {
 
 		delete createdUser.password;
 
-		const token = createToken({ id: createdUser.id});
+		const token = await createToken({ id: createdUser.id});
```
Comment (glowkeeper) [inline src/server/controllers/user.js:28]: Nice :)

**Review by glowkeeper — APPROVED** [review]
Approved

**Pull request merged**